JSON_for_IO-Link.yaml

```diff
@@ -1094,6 +1095,10 @@ paths:
XLO5RwSEhL5+KO1VG5sGTToIQICHEi7hYaG1tnDVS942pk0jh45Su8+vWnZsiXTp79B+/btmTN7HmVlZTVa1ajIUYwf/4plE0BaWhonfzoJQM/f98TNTX9dPYD1cFxX7KXT6QgLa28nxNAwGIo4deoUoaHtHExrXCbleEqNelNKcU+vu2nm0cxSnhEj/sJTTz3h2NB4jaPGjUjAVzfcLHjtPezWLV9ZLU6Y6zwubhatg4Pp379fnafXhbQNwdfXx+Hu3WAoYurUaZw+fRoAd3d3Ro8exYqVy/D3v8Uii4urC7PenEls7BSLkOfOZRITHUNWRQL7nnvuqbHZ41oru7KMdY1EnTt3qTGrrKzcbVu3OpwzPXz4MJnnztttuL3vvddmVcjT04PmzZs79FN7nvQ6twk1gZharVuXlOXH1dWF2GmT6f/A/YBGkaGI6Akx7Nq1u24p/fz8ePyJYXXGFdW7959PpjJlcixZWVmWIWvgQwP553v/oHuPbgQGBfLOO2/zzDPPWB7QlStXiIuL48dDRwDw8PTgrrt6NEoayHpiVlfFd+7cmfCOt1Fz84hG0rpPOXbsWL3XvXo1l1WrVmEymao1AHPoEBER0YTD8P/BroxaylFfHOrm7sb8BfFERUUxd+4ceva6G9C4mH2J1ya+xqFDh2qX0snJiYf/+EebVlrfBTVN4/udu0hMSKSgoMAyPN5xxx2sWv0uH3z4Pg8OfNCS3igqKmbRwkVs+iLZco6hQ4dc09b+umJKR5Leo0Y/h/UO7Mp7LS4uYdbM2aSnZ9Se6ikoZMWKFezauRutWk5WKej/QD86de70/1xI2wxDbWUdPHgQjz46FBcXZ4KDg1nw1t/p8rtOgCLtzFlioieSknLCvpQAXbvezuOPP1bxsEwOPmSN9/71AatXraa0tNRytFWrVoSHd7CZfKxbu5bVq/9huSFPL09GjBzRoJeXlFKUlZVV/ZSWoUz2179N5Sbbz5YZLfIOHjyY7t3vtFSs9dr1nt0/8MLzUWxO3szVq1cxGo0YjUZKSkr48cf/EDsllqVLllO1S71yR7fC3d2Nl156ETc3tyaaPZu4uXaM1xVTKvR6vU1o1r59exIS3yI4JLhitD1JTEyM3V37zmDezjQmKpItW7eSl5vXoKItWJBIUOvWDBv2mN147ZtvtjNz5mxM5SaLNKMjn6NzA3uU3Nxcnh7+NJcv51jOXbldzvqdIID33nuf5OSqXjk8PJx3V72DXq/Hz8+PadOnMfIvz1BQUGg1WTLPEI8cOcaYyCgCgwIsrwenp6dzKvW0VeOzlVmn0xg3/mV69OjRpBpo19lbKkWtSe7GDzlqCtulSxeWLFlM5Ogx5FzO4ch/jhI9IZply5fazMqdreOt6TPeYGLMaxU7th3rLU3lJqbGTqNlS38iIiJsfnvy5M9MmRxLkaHYUsiIfn2JjBzd4HfBlVJcuXKVXzPO2UnSVqWDAIqLim0+FxwcbBOS3HVXD+bFz2HS65MrxLQdIpWCzHMX7ExmtGr5WIVOp+O5Uc8wZkxkk77fXlpSRnJyMr6+vlVFqZYXxCrj061bN8KrbQu02dB0jSmhGoI3MOS6++67WLpsCS9E/ZX8vHx+2LOXmOhXSVyYQMuWLW2l1Ol0PProUNLT01mUuMQq7qq/0AX5hUyZHMvqNavp2DEcgPOZ55n0+qQKOcwPsWOnDsTFxV3XOyKVlWEbz9XV0pXd9NCQoUO4pUULpr8xnZ9PptaSFtPq7Ak8vbyIeXU8I0ZULQg0mZSlZcyYPqvWV0+qH1+1ZqWVlI3cC1pt+bNXXUrVPuPUNI2+fe9j4aIEXhr7MsXFJXy7fQdxM2Yya/ZM/Pz8bLee6PV6XnxxLC/9baxlKHa0FaWdOcvU2KlkZ2dTUFBA/Pz57N93ADChlKJrt9tZuCjRsmfueiY41vHqtbZ0nU5H37738dHHHzFpymsEVq7fOoCrqytP/fkJkpI+ITIyssmFrN7wqr8DZX3c/lbBG7RSpBxLPWmaxoMPPsjceW/i5KQDzO9bLVjwFgUFBfb/QkZJSQlrP1nLm7Pnkp+f36DWNuyJR2kd1JrFi5ZUVKSOwX94iGlvTLuuF+CLior49tvvKnZDN2zOqnfT06/f/bUOr0opsrOzOXz4CPv37ePkzz+TkZ5hWbXQ610JCgqibdu29OzVk27d7qBd23a4uLo0qYjZ2dns27cfVMO16tQp3LID5+LFi+zduw+UwsXVlfvvj0Cv16OUYu/efVy8eAmAtm1DLBu8a6OsrIxt276ivNy8Wtez5z34+5s34qSknOD06dMopfD19aFP7z5oOq2O3r+UXbt2U1hQaFndCgsLq/3PtpjKTRw+coSlS5awZfO2emOI6pMNML8VOH7CK/zpT4/g7e3Nb4Hy8nLLy1KVK1ROTk64u7vj4uIifxfpRmRBVT1JSYPBwMGDB/l0/QY2btyEobCoRtdtvdSn0zkR3vE2nn32GfoPGEBgYMA1/T0ZQaSsF6PRSFZWFr/8kkpa2hmOHTtunsRo4OHRjN916UK70FA6dOhAaGg73N3db8gfgRL+i6UUhBuFjKuCSCkIIqUgUgqCSCmIlIIgUgoipSCIlIIgUgoipSCIlIJIKQgipSBSCoJIKYiUgiBSCoJIKYiUgiBSCiKlIIiUgkgpCCKlIIiUgkgpCCKlIFIKgkgpiJSCIFIKIqUgiJSCIFIKIqUgiJSCSCkIIqUgUgqCSCkIIqUgUgqCSCmIlIIgUgoipSCIlIJIKQgipSCIlIJIKQgipSBSCoJIKYiUgiBSCoJIKYiUgiBSCiKlIIiUgkgpCCKlIFIKgkgpCLXxvz6Kf20HDytIAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDIxLTEwLTA2VDExOjEzOjAxKzAyOjAwSxzPnwAAACV0RVh0ZGF0ZTptb2RpZnkAMjAyMS0xMC0wNlQxMToxMzowMiswMjowMAupbb4AAAAASUVORK5CYII="
```

Comment:
This example is too long for my taste and different to the application/zip answer

Reply:
Good point. I picked a different picture as an example now.